src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -347,9 +352,7 @@
     <Compile Include="System\Drawing\Drawing2D\GraphicsPath.Unix.cs" />
     <Compile Include="System\Drawing\Text\PrivateFontCollection.Unix.cs" />
     <Compile Include="System\Drawing\LibraryResolver.cs" />
-    <Compile Include="misc\ExternDll.Unix.cs" />
-    <Compile Include="$(CommonPath)Interop\Unix\Interop.Libraries.cs"
-             Link="Common\Interop\Unix\Interop.Libraries.cs" />
+    <Compile Include="Interop\Unix\Interop.Libraries.cs" />
```

Comment:
Could you also please update <TargetFrameworks> to not target Unix? I think we effectively support this library on Linux and macOS only not on any Unix* RIDs like tvOS or solaris.

Reply:
Yep, which is not wrong because the library is not supported on "Unix". I'll open another issue if you don't want to mix the fix with this change

Reply:
> Yep, which is not wrong because the library is not supported on "Unix".

You mean that cause Unix would imply iOS/tvOS and others we don't support right?

> I'll open another issue if you don't want to mix the fix with this change

Sounds good. I rather keep this PR as is and have another issue to fix this.